tests/test_clicks.py

```diff
@@ -0,0 +1,115 @@
+from poprox_storage.repositories.clicks import DbClicksRepository
+from poprox_concepts import Account
+from datetime import datetime
+
+import json
+import pytest
+import os
+from sqlalchemy import create_engine, text
+from uuid import uuid4
+
+db_password = os.environ.get("POPROX_DB_PASSWORD", "")
+db_port = os.environ.get("POPROX_DB_PORT", "")
+
+DEFAULT_PG_URL = f"postgresql://postgres:{db_password}@127.0.0.1:{db_port}/poprox"
+
+
+@pytest.fixture(scope="session")
+def pg_url():
+    """
+    Provides base PostgreSQL URL for creating temporary databases.
+    """
+    return os.getenv("CI_POPROX_PG_URL", DEFAULT_PG_URL)
+
+
+def test_get_active_survey(pg_url: str):
+    engine = create_engine(pg_url)
+    with engine.connect() as conn:
+        conn.execute(text("delete from clicks;"))
+        conn.execute(text("delete from newsletters;"))
+        conn.execute(text("delete from articles;"))
+
+        dummy_click_1 = uuid4()
```

Comment:
We know these are for testing but it's not obvious in the code below that these are the ids and not the click objects (for example), so I'd probably name these something that includes `id` like:
```
click_id_1 = ...
user_id_1 = ...
article_id_1 = ...
```